content/rs/release-notes/rs-6-4-2-releases/rs-6-4-2-30.md

```diff
@@ -152,6 +152,10 @@ Certain operating systems, such as RHEL 8, have already removed support for 3DES
 
 ## Known limitations
 
+### Feature limitations
+
+- RS101204 - High memory consumption caused by the `persistence_mgr` service when AOF persistence is configured for every second.
```

Comment:
```suggestion
- RS101204 - High memory consumption caused by the `persistence_mgr` service when AOF persistence is configured for every second. Monitor RAM usage of the process. In case of high usage, the temporary workaround is to restart the service by running `supervisorctl restart persistence_mgr`. A permanent fix is to install or upgrade to the 6.4.2 June maintenance release.
```